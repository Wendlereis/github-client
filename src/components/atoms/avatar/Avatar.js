import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

const StyledFigure = styled.img`
  border-radius: 50%;
  border: 5px solid #fafafa;
  height: inherit;
  box-sizing: border-box;
`

export default function Avatar(props) {
  return <StyledFigure {...props} />
}

Avatar.propTypes = {
  src: PropTypes.string.isRequired
}
