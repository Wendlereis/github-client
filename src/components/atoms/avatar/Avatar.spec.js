import React from 'react'
import { shallow } from 'enzyme'
import Avatar from './Avatar'

describe('Avatar', () => {
  it('should render correctly with an image', () => {
    const component = shallow(
      <Avatar src="https://avatars.dicebear.com/v2/identicon/.svg" />
    )

    expect(component).toMatchSnapshot()
  })
})
