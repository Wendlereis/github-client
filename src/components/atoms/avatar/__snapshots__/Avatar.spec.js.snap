// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render correctly with an image 1`] = `
<styled.img
  src="https://avatars.dicebear.com/v2/identicon/.svg"
/>
`;
